lib/stub_ui/lib/src/ui/window.dart

```diff
@@ -35,6 +35,71 @@ typedef PlatformMessageResponseCallback = void Function(ByteData data);
 typedef PlatformMessageCallback = void Function(
     String name, ByteData data, PlatformMessageResponseCallback callback);
 
+/// Time-related performance metrics of a frame.
```

Comment:
this needs much more documentation. For example, how to get one of these, whether it makes sense to examine the data at all in debug mode, what it measures, etc.